source/ocsp-support/ocsp-support.rst

```diff
@@ -339,8 +364,10 @@ a response and the OCSP responder is down.
 Drivers that fail “Soft Fail Test” MUST document that their driver’s
 TLS library utilizes “hard fail” behavior in the case of an
 unavailable OCSP responder in contrast to the mongo shell and drivers
-that utilize “soft-fail” behavior. Such drivers MUST also document the
-potential backwards compatibility issues as noted in the `Backwards
+that utilize “soft fail” behavior. They also MUST document the change
```

Comment:
Please replace with non-directional double quote characters. Bonus points if you catch any other instances of this in the document (I see a few in the lines above).

Reply:
Done, with the bonus points :)  I think they were probably left over when this was ported from google docs. Out of curiosity, why is having directional quotes considered undesirable?